hawkular-tenant-jaxrs-filter/src/main/java/org/hawkular/jaxrs/filter/tenant/TenantFeature.java

```diff
@@ -24,6 +24,9 @@
 import javax.ws.rs.ext.Provider;
 
 /**
+ * A JAX-RS dynamic feature that adds the {@link TenantFilter} to all methods which have {@link TenantRequired}
+ * effectively as "true".
```

Comment:
when TenantRequired is not specified at all, from https://github.com/hawkular/hawkular-commons/pull/72/files#diff-112e3d7f5467ae0e9ba60c0a73f8f51eR36 I understand it is still considered as required. If that's the case, the above explanation can be misleading to think that TenantRequired must be declared.